### PR TITLE
Feat/boarding pass

### PR DESCRIPTION
**Pull request by Parle648**
## Describe your changes

- Created boarding pass verification pages and link them

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://codecraftersintership.atlassian.net/browse/SCRUM-147?atlOrigin=eyJpIjoiYmIzMzk4OWI4ZGYwNDkzOWIwYjMyMmJkNzJiNTQ5NzciLCJwIjoiaiJ9)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/ActionBtns/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  const { verifyRequest, isVerified, isOngoing, isVerifiedSuccessfully } =
+    useVerifyRequest();
+
+  return (
+    <Box sx={isOngoing ? actionBtnsContainerLoading : actionBtnsContainer}>
+      {!isOngoing && (
+        <>
+          <Button
+            sx={backButtonStyles}
+            label='Back'
```
Comment (AlexeyBezrebriy) [inline src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/ActionBtns/index.tsx]: i18n

File: src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/BoardingPassInformBlock/components/TicketModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  children: ReactNode;
+}
+
+export const TicketModal: FC<TicketModalInterface> = ({
+  ticketImageLink,
+  children,
+}) => {
+  return (
+    <Box
+      sx={{
```
Comment (AlexeyBezrebriy) [inline src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/BoardingPassInformBlock/components/TicketModal/index.tsx]: 1.3.4

File: src/pages/PickUpTheOrder/BoardingPass/IdentityVerification/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+export const IdentityVerification: FC = () => {
+  const [disabled, setIsDisabled] = useState<boolean>(false);
+  const { user } = useAppSelector((store) => store.auth);
+
+  return (
+    <>
+      <Header
+        hasBackIcon
+        pageName='Identity Verification'
```
Comment (AlexeyBezrebriy) [inline src/pages/PickUpTheOrder/BoardingPass/IdentityVerification/index.tsx]: i18

File: src/pages/PickUpTheOrder/BoardingPass/PreArrivalPage/components/CheckList/index.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            <li key={itemKey} style={listItemStyles}>
+              {t(itemKey)}
+            </li>
+          ))}
+        </ol>
+      </Box>
+      <Typography sx={warningParagraph}>
+        {t(
+          'checklist.warning',
+          'Confirm you lock the transport vehicle securely, ensuring safety as required by the LBA.'
```
Comment (AlexeyBezrebriy) [inline src/pages/PickUpTheOrder/BoardingPass/PreArrivalPage/components/CheckList/index.tsx]: i18

File: src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/BoardingPassInformBlock/components/TicketModal/index.tsx (hunk 2 above)
Comment (Parle648) [reply src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/BoardingPassInformBlock/components/TicketModal/index.tsx]: done

File: src/pages/PickUpTheOrder/BoardingPass/IdentityVerification/index.tsx (hunk 3 above)
Comment (Parle648) [reply src/pages/PickUpTheOrder/BoardingPass/IdentityVerification/index.tsx]: done

File: src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/ActionBtns/index.tsx (hunk 1 above)
Comment (Parle648) [reply src/pages/PickUpTheOrder/BoardingPass/BoardingPassVerificationPage/components/ActionBtns/index.tsx]: done

File: src/pages/PickUpTheOrder/BoardingPass/PreArrivalPage/components/CheckList/index.tsx (hunk 4 above)
Comment (Parle648) [reply src/pages/PickUpTheOrder/BoardingPass/PreArrivalPage/components/CheckList/index.tsx]: done

File: src/pages/PickUpTheOrder/BoardingPass/IdentityVerification/components/CustomerInformBlock/styles.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
+:root {
+    --white: #fff;
+    --gray: #575757b0;
+    --purple: #7367F0;
+}
```
Comment (oyakovytskyi) [inline src/pages/PickUpTheOrder/BoardingPass/IdentityVerification/components/CustomerInformBlock/styles.css]: Is this necessary here?

Comment (Parle648) [reply src/pages/PickUpTheOrder/BoardingPass/IdentityVerification/components/CustomerInformBlock/styles.css]: nope just forget to delete it

**Review by oyakovytskyi — APPROVED** [review]

**Review by AlexeyBezrebriy — APPROVED** [review]

**Pull request merged**